cinnabar/wrangle.py

```diff
@@ -58,85 +65,35 @@ def toDF(self):
-            # Data is already numeric
-            error = expt_dDG
-        self.dDG = float(error)
-
-
-class FEResults(object):
-    """
-    Wrapper class to store results from Free Energy calculations.
-
-    Useful for programatically creating a container to pass to FEMap class.
-    """
-
-    def __init__(self):
-        """Create results object"""
-        # Initializing attributes
-        self.experimental_results = dict()
-        self.calculated_results = list()
-        self.fe_results = {
-            "Experimental": self.experimental_results,
-            "Calculated": self.calculated_results,
-        }
-
-    def add_experimental_obj(self, experimental_result):
-        """Adds experimental result from ExperimentalResult object."""
-        self.experimental_results[experimental_result.ligand] = experimental_result
-
-    def add_calculated_obj(self, calculated_result):
-        """Adds calculated result from CalculatedResult object."""
-        self.calculated_results.append(calculated_result)
-
-    def add_experimental_result(self, ligand, expt_value, expt_error):
-        """
-        Add experimental result from plain explicit values.
+        self.DG = expt_DG
+        self.dDG = expt_dDG
 
-        Examples
-        --------
-        >>> fe_results = FEResults()
-        >>> fe_results.add_experimental_result("CAT-13a", -8.93, 0.10)
-        """
-        self.experimental_results[ligand] = ExperimentalResultFromCSV(
-            ligand, expt_value, expt_error
-        )
 
-    def add_calculated_result(self, ligand_a, ligand_b, calc_value, mbar_error, other_error):
-        """
-        Add calculated relative result from plain explicit values.
+class FEMap:
+    results: list[Union[RelativeResult, ExperimentalResult]]
+    computational_graph: nx.MultiDiGraph
+    experimental_graph: nx.MultiDiGraph
 
-        Examples
-        --------
-        >>> fe_results = FEResults()
-        >>> fe_results.add_calculated_result("CAT-13a", "CAT-17g", 0.36, 0.11, 0.0)
-        """
-        self.calculated_results.append(
-            RelativeResultFromCSV(ligand_a, ligand_b, calc_value, mbar_error, other_error)
-        )
+    _name_to_id: dict[str, int]
+    _id_to_name: dict[int, str]
 
-
-class FEMapFromCSV(object):
-    def __init__(self, input_data):
+    def __init__(self, input_data: list[Union[RelativeResult, ExperimentalResult]]):
         """
         Construct Free Energy map of simulations from input data.
 
         Parameters
         ----------
-            input_data: csv file path or path-like object or dict-like object
-                File path to csv file or python dict-like object.
+            input_data: dict-like object
+                dict-like object.
 
         Examples
         --------
         To read from a csv file specifically formatted for this, you can use:
 
-        >>> fe = wrangle.FEMap('../data/example.csv')
+        >>> fe = wrangle.FEMap.from_csv('../data/example.csv')
```

Comment:
I think we just need this break where we make this constructor a classmethod, but it already doesn't handle absolute computational results so maybe it'd be kinder to just kill it